### PR TITLE
Development balance view

### PR DESCRIPTION
**Pull request by fnscoder**
Created a tests directory inside app core directory
Created a file of view balance tests
Implemented view balance

### CODE REVIEW

File: core/tests/test_view_balance.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            installment=85.6
+        )
+        loan_paid.save()
+
+        for payment in range(12):
+            p = Payment(
+                loan=loan_paid,
+                user=user,
+                payment='made',
+                date=datetime.strptime('10052019', '%d%m%Y'),
```
Comment (felipelfb) [inline core/tests/test_view_balance.py]: Payments would have the same date, which should not be accepted.

File: core/tests/test_view_balance.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                amount=85.6,
+            )
+            p.save()
+
+        self.response_invalid = self.client.post('/loans/0/balance/')
```
Comment (felipelfb) [inline core/tests/test_view_balance.py]: We can use GET instead of POST

**Review by marcelomileris — APPROVED** [review]

File: core/tests/test_view_balance.py (hunk 2 above)
Comment (fnscoder) [reply core/tests/test_view_balance.py]: Changed

File: core/tests/test_view_balance.py (hunk 1 above)
Comment (fnscoder) [reply core/tests/test_view_balance.py]: Changed

**Pull request merged**